ScheduleCreator/controllers/CoursesController.java

```diff
@@ -26,56 +25,72 @@
  */
 
 public class CoursesController implements Initializable {
-    
+	@FXML private Button semesterButton;
+	@FXML private ComboBox<String> semesterComboBox;    
   
-    @FXML private ComboBox<String> comboBox;    
+	@FXML private ComboBox<String> courseComboBox;    
 
-    @FXML private ListView selectedCourses;
+	@FXML private ListView selectedCourses;
     
-    @FXML private Button courseButton;
-    @FXML private Button removeCourseButton;
+	@FXML private Button courseButton;
+	@FXML private Button removeCourseButton;
 
-    @Override
-    public void initialize(URL url, ResourceBundle rb) {
-        try {
-        loadAllCourses();
-        loadSelectedCourses();
-        }
-        catch (Exception e) {};
+	@Override
+	public void initialize(URL url, ResourceBundle rb) {
+		try {
+			loadSemesters();
+		}
+		catch (Exception e) {};
         
-    }
-    
-    public void addSelectedCourse(ActionEvent _event) throws Exception {
-        String choice = comboBox.getValue();
-        List<String> courseList = new ArrayList();
-        courseList.add(choice);
-        selectedCourses.getItems().add(choice);
-        DBAdapter.saveCourse(choice);
-    }
+	}
     
-    public void removeSelectedCourse(ActionEvent _event) throws Exception {
-        Object itemToRemove = selectedCourses.getSelectionModel().getSelectedItem();
-        String courseToDelete = (String)itemToRemove;
-        selectedCourses.getItems().remove(itemToRemove);
-        DBAdapter.removeCourse(courseToDelete);
-    }
+	public void addSelectedCourse(ActionEvent _event) throws Exception {
+		String choice = courseComboBox.getValue();
+		List<String> courseList = new ArrayList();
+		courseList.add(choice);
+		selectedCourses.getItems().add(choice);
+		DBAdapter.saveCourse(choice);
+	}
+	public void switchSemester(ActionEvent _event) throws Exception {
+		clearCalendar();
+		clearSectionList();
+                String[] temp = semesterComboBox.getValue().split(" ");
+                String semester = temp[0].toLowerCase() + temp[1];
```

Comment:
What if we move `ScheduleCreator/resources/raw/spring2020courses.txt` into `ScheduleCreator/resources/raw/courslists/Spring 2020`? Then we can just directly use the semester name as an argument to getCourses()

Reply:
ScheduleCreator/resources/raw/courslists/Spring 2020
is this a file or a directory?

Reply:
I don't really see what you mean but I'm sure whatever you're suggesting works better

Reply:
It's a file. I just used the filenames in DBAdapter.getSemesters() under the `semesters` directory, we could do the same thing for getCourses()  so you don't have to work around the semester names returned by getSemesters()

Reply:
Eh it doesn't work better right now, just looking forward to when we have to replace all of these 
```java
        File file = new File("src/ScheduleCreator/resources/raw/whatever");
```
lines because of the classpath thing.

Reply:
Isnt that what the Boilerplate class is for?

Reply:
Yes, but those methods will have to be replaced too because I haven't figured out how to access classpath files. To see what I mean, visit the `dist` directory and run `Schedule-Creator.jar` - all of the files we referenced don't exist on the user's hard drive so those lines don't work.

Reply:
> It's a file. I just used the filenames in DBAdapter.getSemesters() under the `semesters` directory, we could do the same thing for getCourses() so you don't have to work around the semester names returned by getSemesters()

Ohhh, I see.  That would probably be better.  I only did that to try to stick to some naming convention for the txt files but it does make it more complicated so it can definitely be changed.

Reply:
I'll create an issue for it. We don't have to worry about it now.